lib/Runtime/Base/ScriptContext.cpp

```diff
@@ -1591,7 +1591,10 @@ if (!sourceList)
 
         bool isLibraryCode = ((loadScriptFlag & LoadScriptFlag_LibraryCode) == LoadScriptFlag_LibraryCode);
 
-        bool utf8FromExternal = false;
+        // We assume that any buffer that wasn't passed in with LoadScriptFlag_Utf8Source is UTF16-LE
```

Comment:
I think you can just revert this file to the version prior to my change. That'll create less noise for future merging to master.